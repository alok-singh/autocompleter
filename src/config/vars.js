// api debounce timein milliseconds
export const API_DEBOUNCE_TIME = 500;

// autocompleter api. can omit api key from here and put in environment variables
export const AUTOCOMPLETER_API_URL = 'http://www.omdbapi.com/?apikey=c4267e95&s=${KEYWORD}&type=movie';

// a random fallback image can be set according to brand
export const FALLBACK_IMAGE = 'http://www.omdbapi.com/src/poster.jpg';