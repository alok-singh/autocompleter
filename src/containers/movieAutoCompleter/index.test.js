import React from 'react';
import { render } from '@testing-library/react';
import AutoCompleter from './index';
import { API_DEBOUNCE_TIME, AUTOCOMPLETER_API_URL } from "../../config/vars";

test('renders learn react link', () => {
  expect(render(<AutoCompleter
    debounceTime={API_DEBOUNCE_TIME}
    apiUrl={AUTOCOMPLETER_API_URL}
    dataModel="searchAPIParser"
  />).baseElement).toMatchSnapshot();
});
