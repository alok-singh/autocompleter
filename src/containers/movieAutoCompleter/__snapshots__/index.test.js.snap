// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<body>
  <div>
    <section
      class="movie-autocompleter-container"
    >
      <div
        class="tag-input-wrapper"
      >
        <input
          class="u-input"
          value=""
        />
      </div>
    </section>
  </div>
</body>
`;
